docs/developer.md

```diff
@@ -36,6 +36,18 @@ For an in-depth look at MDC-Web's architecture, check out [architecture.md](./ar
 
 Check out our `webpack.config.js` for more details!
 
+### Deployment
+
+Demo pages of Material Components for Web are deployed in Google Cloud. Request an access to project id: material-components-web to get deployment permissions to app engine.
+
+    # Login to Google Cloud account and choose material-components-web project id.
+    gcloud init
```

Comment:
Done.

Also renamed the section to 'Hosting' and moved gcloud commands to Contributing doc with some note.